crates/html_node/src/lib.rs

```diff
@@ -43,6 +49,93 @@ struct MinifyTask {
 pub struct MinifyOptions {
     #[serde(default)]
     filename: Option<String>,
+
+    // Parser options
+    #[serde(default)]
+    iframe_srcdoc: bool,
+    #[serde(default)]
+    scripting_enabled: bool,
+
+    // Minification options
+    #[serde(default)]
+    force_set_html5_doctype: bool,
+    #[serde(default = "default_collapse_whitespaces")]
+    collapse_whitespaces: CollapseWhitespaces,
+    // Remove safe empty elements with metadata content, i.e. the `script` and `style` element
+    // without content and attributes, `meta` and `link` elements without attributes and etc
+    #[serde(default = "true_by_default")]
+    remove_empty_metedata_elements: bool,
+    #[serde(default = "true_by_default")]
+    remove_comments: bool,
+    #[serde(default = "default_preserve_comments")]
+    preserve_comments: Option<Vec<CachedRegex>>,
+    #[serde(default = "true_by_default")]
+    minify_conditional_comments: bool,
+    #[serde(default = "true_by_default")]
+    remove_empty_attributes: bool,
+    #[serde(default = "true_by_default")]
+    remove_redundant_attributes: bool,
+    #[serde(default = "true_by_default")]
+    collapse_boolean_attributes: bool,
+    #[serde(default = "true_by_default")]
+    normalize_attributes: bool,
+    #[serde(default = "minify_json_by_default")]
+    minify_json: MinifyJsonOption,
+    #[serde(default = "minify_js_by_default")]
+    minify_js: MinifyJsOption,
+    #[serde(default = "minify_css_by_default")]
+    minify_css: MinifyCssOption,
+    #[serde(default)]
+    minify_additional_scripts_content: Option<Vec<(CachedRegex, MinifierType)>>,
+    #[serde(default)]
+    minify_additional_attributes: Option<Vec<(CachedRegex, MinifierType)>>,
+    #[serde(default = "true_by_default")]
+    sort_space_separated_attribute_values: bool,
+    #[serde(default)]
+    sort_attributes: bool,
+
+    // Codegen options
+    #[serde(default)]
+    tag_omission: Option<bool>,
+    #[serde(default)]
+    self_closing_void_elements: Option<bool>,
+}
+
+const fn true_by_default() -> bool {
```

Comment:
Maybe we should export from `swc_html_minifier` to avoid copy/pasting, what do you think (same for below)?

Reply:
I think copy/pasting is better

Reply:
But I think we should export config struct from `swc_html_minifier`